projects/grandine-windows-support.md

```diff
@@ -0,0 +1,70 @@
+# Grandine Windows Support
+
+To support building Grandine on Windows.
```

Comment:
I understand windows is a priority considering its adoption but in terms of cross platform support, you could consider adding more stuff like macos, bsd or other architectures like arm, riscv